roles/mailserver/files/docker-compose.yml

```diff
@@ -138,6 +166,13 @@ services:
       - mailserver
       - postfixadmin
       - rainloop
+{% if use_opencart %}
+      - /mnt/docker/opencart:/opencart
```

Comment:
This is not right. This is probably why the nginx container is not getting created.  This section is for cross-container dependency for startup, not volume links.

Reply:
containers coming up now, there were some config issues with the opencart.conf for nginx.... though its not pretty but i can see the cart now.